scripts/feature_tests/upgrade/upgrade_common.sh

```diff
@@ -0,0 +1,448 @@
+#!/bin/bash
+
+export IMAGE_OS=${IMAGE_OS:-"Ubuntu"}
+export IMAGE_USERNAME=${IMAGE_USERNAME:-"metal3"}
+export KUBERNETES_VERSION="v1.18.0"
+export KUBERNETES_BINARIES_VERSION="v1.18.0"
+export UPGRADED_K8S_VERSION_1="v1.18.1"
+export UPGRADED_K8S_VERSION_2="v1.18.2"
+export UPGRADED_BINARY_VERSION="v1.18.1"
+export CLUSTER_APIENDPOINT_IP=${CLUSTER_APIENDPOINT_IP:-"192.168.111.249"}
+export NUM_NODES=${NUM_NODES:-"4"}
+
+export CAPM3RELEASE=${CAPM3RELEASE:-"v0.3.0"}
+export CAPIRELEASE=${CAPIRELEASE:-"v0.3.4"}
+
+export IMAGE_URL=${IMAGE_URL:-"http://172.22.0.1/images/bionic-server-cloudimg-amd64-raw.img"}
+export IMAGE_CHECKSUM=${IMAGE_CHECKSUM:-"http://172.22.0.1/images/bionic-server-cloudimg-amd64-raw.img.md5sum"}
+
+# The next lines to be fixed in the CI
+export CAPM3PATH="/home/ubuntu/go/src/github.com/metal3-io/cluster-api-provider-metal3"
```

Comment:
this should be a default only, and you should use https://github.com/metal3-io/metal3-dev-env/blob/master/lib/common.sh#L96 otherwise you might hit an issue if it is changed (the repo would not be cloned there)

Reply:
done